.github/workflows/release-with-github.yml

```diff
@@ -43,16 +43,8 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - name: Install Rust
-        uses: dtolnay/rust-toolchain@stable
-      - name: Install dependencies
-        run: |
-          wget https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz
-          tar -xvzf cargo-binstall-x86_64-unknown-linux-musl.tgz
-          rm cargo-binstall-x86_64-unknown-linux-musl.tgz
-          mv cargo-binstall $HOME/.cargo/bin
-          cargo binstall cargo-release -y
-          cargo binstall ripgrep -y
+      - uses: cargo-bins/cargo-binstall@main
+      - run: cargo binstall cargo-release ripgrep -y
```

Comment:
Does this workflow use some special features of ripgrep that grep doesn't have?  If not, we should use grep.

Reply:
Capture groups, which AFAIK, work differently on MacOS and Linux. Installing both packages took 1.983274393s. I'd appreciate it if we could leave it as is for the sake of debuggability